build-tools/packages/build-tools/src/test/tsconfig.json

```diff
@@ -52,9 +50,11 @@
 		// "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
 		// "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
 		"incremental": true,
+		// "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
+		"esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
+		/* Basic Options */
```

Comment:
Basic Options banner moved - banners are not preserved. Those should be reviewed 1 by 1.
I was also thinking that target is pretty key and may deserve to be sorted early. lib might be in the same bucket.

Reply:
I think the banner moving is a bug in sort-jsonc. But I don't know if it could realistically be fixed since the comment has to anchor to a  "real node" in the JSON. In this case it gets anchored to "target" so it moves with it. (Sorry if this was obvious). That seems reasonable behavior but not ideal.

Reply:
I would expect that it moves. I'm just saying that with sorting applied changes will have to be reviewed. The sort list perhaps won't always have groups like tsconfigs in individual packages.
I don't expect we'll have many such cases going foward.